python/tk_maya/menu_generation.py

```diff
@@ -112,38 +109,40 @@ def _add_context_menu(self):
         # create the menu object
         # the label expects a unicode object so we cast it to support when the context may
         # contain info with non-ascii characters
-        ctx_menu = pm.subMenuItem(
-            label=ctx_name.decode("utf-8"), parent=self._menu_handle
+        ctx_menu = cmds.menuItem(
+            label=six.ensure_str(ctx_name), parent=self._menu_path, subMenu=True
         )
 
         # link to UI
-        pm.menuItem(
-            label="Jump to Shotgun", parent=ctx_menu, command=Callback(self._jump_to_sg)
+        cmds.menuItem(
+            label="Jump to Shotgun", parent=ctx_menu, command=self._jump_to_sg
         )
 
         # Add the menu item only when there are some file system locations.
         if ctx.filesystem_locations:
-            pm.menuItem(
-                label="Jump to File System",
-                parent=ctx_menu,
-                command=Callback(self._jump_to_fs),
+            cmds.menuItem(
+                label="Jump to File System", parent=ctx_menu, command=self._jump_to_fs,
             )
 
         # divider (apps may register entries below this divider)
-        pm.menuItem(divider=True, parent=ctx_menu)
+        cmds.menuItem(divider=True, parent=ctx_menu)
 
         return ctx_menu
 
-    def _jump_to_sg(self):
+    def _jump_to_sg(self, state):
```

Comment:
As with other places, we now get the state of the menu coming through to the callback, even though we don't need it.